src/photos.cs

```diff
@@ -302,8 +310,8 @@ interface PHAssetContentEditingInputExtensions {
 
 	[iOS (8,0)]
 	[TV (10,0)]
-	[NoMac]
-	[BaseType (typeof (NSObject))]
+	[Mac (10,13, onlyOn64: true)]
```

Comment:
sure about 10,13 ? other related types are 10,15
if it's only in xcode11 headers assume it's 10,15 - unless tested